autotst/conformer/systematic.py

```diff
@@ -213,25 +217,42 @@ def opt_conf(conformer, calculator, i):
 
             calculator.atoms = conformer.ase_molecule
 
-        from ase.constraints import FixBondLengths
-        c = FixBondLengths(labels)
-        conformer.ase_molecule.set_constraint(c)
-
         conformer.ase_molecule.set_calculator(calculator)
-
         opt = BFGS(conformer.ase_molecule, logfile=None)
```

Comment:
Switch `BFGS` to `LBFGS` at the recommendation of Eric Hermes. The import at the top will need to change too.